### ISSUE TITLE
"Saw" type item duplication

### ISSUE THREAD

**Issue by sharkweekthings** [PrinceOfAmber/Storage-Network#547]
Minecraft Version: 
1.19.2
Forge Version: 
43.3.2
Mod Version: 
SimpleStorageNetwork-1.19.2-1.7.0 & silent-gear-1.19.2-3.2.2
Single Player or Server:
Both
Describe problem (what you were doing; what happened; what should have happened):
Item Duplication w/ "Saw" type items when following the steps in the attached video. 

This is an issue occurring on my Modpack with these two mod interactions.

Log file link:

Video/images/gifs (direct upload or link): 

https://github.com/Lothrazar/Storage-Network/assets/134178249/2bbaa166-1a79-4d5b-94a7-a1a7bb509ef7

**Comment by Lothrazar** [comment]
yes im also seeing this on minecraft 1.20.1 . if im running a s table 120 fps, then click sort, it dips down to 80 frames for a second or two before jumping back up.
